fairseq/models/speech_to_text/xm_transformer.py

```diff
@@ -632,7 +632,7 @@ def build_model(cls, args, task):
 
         # make sure all arguments are present in older models
         base_architecture(args)
-        if getattr(args, "load_pretrained_decoder_from", None):
+        if getattr(args, "load_pretrained_decoder_from", None) is not None:
```

Comment:
nit: is not None condition check is not needed I guess?
```suggestion
        if getattr(args, "load_pretrained_encoder_from", None):
```